assets/app/view/create_game.rb

```diff
@@ -44,26 +44,45 @@ def render_content
     end
 
     def render_inputs
+      @min_p = {}
+      @max_p = {}
+
       games = (Lib::Params['all'] ? Engine::GAMES : Engine::VISIBLE_GAMES).map do |game|
+        @min_p[game.title] = game::CERT_LIMIT.keys.min
+        @max_p[game.title] = game::CERT_LIMIT.keys.max
+
         title = game.title
         title += " (#{game::GAME_LOCATION})" if game::GAME_LOCATION
-        if game::DEV_STAGE == :production
-          h(:option, { attrs: { value: game.title } }, title)
-        else
-          h(:option, { attrs: { value: game.title } }, "#{title} (#{game::DEV_STAGE})")
-        end
+        title += " (#{game::DEV_STAGE})" if game::DEV_STAGE != :production
+
+        h(:option, { attrs: { value: game.title } }, title)
       end
-      h(:div, [
-        render_input('Game Title', id: :title, el: 'select', children: games),
-        render_input('Description', id: :description),
+
+      limit_range = lambda do
+        min = Native(@inputs[:max_players]).elm.min = @min_p[Native(@inputs[:title]).elm.value]
```

Comment:
why call Native(@inputs) n times, store that first and then you can use that

Reply:
… because sometimes I don’t see the wood for the trees. Fixed.